File: CS/NavBarExample/ViewModel.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Data;

namespace NavBarExample
{
    public class ViewModel
    {
        public ObservableCollection<Employee> Employees
        {
            get;
            set;
        }

        public ViewModel()
        {
            Employees = new ObservableCollection<Employee>() {
            new Employee(){Name="Alex", Department="Management"},
            new Employee(){Name="Paul", Department="Logistics"},
            new Employee(){Name="Peter", Department="Logistics"},
            new Employee(){Name="Mary", Department="Management"},
            new Employee(){Name="John", Department="Sales"},
            new Employee(){Name="Mark", Department="Sales"},
            new Employee(){Name="Kumar", Department="Advertisement"},
            new Employee(){Name="Janet", Department="Logistics"},
            new Employee(){Name="Tom", Department="Advertisement"},};
        }
    }

    public class Employee
    {
        public string Name
        {
            get;
            set;
        }
        public string Department
        {
            get;
            set;
        }
    }
}

File: VB/NavBarExample/ViewModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Collections.ObjectModel
Imports System.ComponentModel
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Data

Namespace NavBarExample
    Public Class ViewModel
        Public Property Employees() As ObservableCollection(Of Employee)

        Public Sub New()
            Employees = New ObservableCollection(Of Employee)() From { _
                New Employee() With {.Name="Alex", .Department="Management"}, _
                New Employee() With {.Name="Paul", .Department="Logistics"}, _
                New Employee() With {.Name="Peter", .Department="Logistics"}, _
                New Employee() With {.Name="Mary", .Department="Management"}, _
                New Employee() With {.Name="John", .Department="Sales"}, _
                New Employee() With {.Name="Mark", .Department="Sales"}, _
                New Employee() With {.Name="Kumar", .Department="Advertisement"}, _
                New Employee() With {.Name="Janet", .Department="Logistics"}, _
                New Employee() With {.Name="Tom", .Department="Advertisement"} _
            }
        End Sub
    End Class

    Public Class Employee
        Public Property Name() As String
        Public Property Department() As String
    End Class
End Namespace
